packages/manager/modules/account-migration/src/notification/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,20 @@
+{
+  "ovh_manager_account_migration_pending_contracts1": "Nous avons modifié nos Conditions d’utilisation. Vous avez jusqu’au {{date}} pour",
```

Comment:
Could all these translations not be split ?

Reply:
Marie already pointed this out. But the problem is, we have couple of links in between the sentence, which will also have tracking. Hence I am not able to find a way to accommodate everything in a single sentence, without splitting... Please suggest if you have alternative solutions :)